app/static/notes.css

```
@@ -0,0 +1,352 @@
+    --paper-bg: #2d2d2d;
+    --paper-line: #3d3d3d;
+    --text-color: #e0e0e0;
+    --toolbar-bg: #333;
+}
+
+.notes-layout {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+    background-color: #f0f2f5;
+    font-family: 'Lato', sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Lato" (font-family-name-quotes)

(font-family-name-quotes)